README.md

```diff
@@ -162,11 +162,13 @@ Sharing in Tomcat.
 See instructions [here](http://enable-cors.org/server_tomcat.html) for Tomcat 7 and [here](https://www.dforge.net/2013/09/16/enabling-cors-on-apache-tomcat-6/) for Tomcat 6.
 
 ## Usage
-Before using the Order Entry Open Web Application, the administrator should make the following one time configurations using the admin portal. 
+Before using the Order Entry Open Web Application, the administrator should make the following one time configurations using the admin portal.
 1. Create an Encounter Type called `Order Entry` or any other name.
 2. Create a setting `order.encounterType` whose value should correspond to the name given to the Encounter Type created above.
 3. Create an Encounter Role called `Clinician` or any other name.
 4. Create a setting `order.encounterRole` whose value should correspond to the name given to the encounter role created in 3 above.
+5. Create a date format, setting `orderentryowa.dateAndTimeFormat` as the name, with a sample value of `DD-MMM-YYYY HH:mm`, in the global properties
```

Comment:
This is not as clear as the one you made below.

Reply:
@dkayiwa I've corrected it